### PR TITLE
Support for PHP 7

### LINKED ISSUE #526: Support PHP 7
http://php.net/manual/en/migration70.incompatible.php
- [x] Validate use of indirect variables/properties/methods (e.g. $$thing)
- [x] Foreaches by-reference where necessary
- [x] Use of mcrypt aliases
- [x] Fixes to Custom Session Handler Return Values

### PR DESCRIPTION
**Pull request by LloydW93**
Fixes #526. Turned out that PHP 7 itself was a case of Just Works, but Twig 2.0 was a real pain.

### CODE REVIEW

File: src/Controllers/Setup/checks.php [hunk 1]
```diff
@@ -247,16 +240,10 @@ function convertPHPSizeToBytes($sSize)
...
             <code>
-              apt-get install php5-curl php5-geoip php5-gd php5-ldap php5-mcrypt php5-pgsql php5-dev php-pear<br>
-              pear channel-discover pear.twig-project.org<br>
-              pear install twig/Twig<br>
-              pear install twig/CTwig<br>
-              echo "extension=mcrypt.so" > /etc/php5/mods-available/mcrypt.ini<br>
-              ln -s /etc/php5/mods-available/mcrypt.ini /etc/php5/apache2/conf.d/20-mcrypt.ini<br>
-              echo "extension=twig.so" > /etc/php5/mods-available/twig.ini<br>
-              ln -s /etc/php5/mods-available/twig.ini /etc/php5/apache2/conf.d/20-twig.ini<br>
+              apt install php-curl php-geoip php-gd php-ldap php-pgsql php-dev composer<br>
```
Comment (LloydW93) [inline src/Controllers/Setup/checks.php]: Need to add php-mbstring

**Comment by LloydW93** [comment]
Nope :(

File: .travis.yml [hunk 2]
```diff
@@ -1,8 +1,14 @@
 language: php
 php:
     - '5.6'
```
Comment (LloydW93) [inline .travis.yml]: We need to drop 5.6 as Twig 2 only supports 7.

Comment (LordAro) [inline .travis.yml]: yeah... guess there's absolutely no way to support both? be a shame to completely drop 5.6

**Comment by AJDurant** [comment]
Still need to make the `apt install` line in checks.php match up

**Pull request merged**